pylint/checkers/typecheck.py

```diff
@@ -1227,20 +1227,25 @@ def _check_uninferable_call(self, node):
             # Decorated, see if it is decorated with a property.
             # Also, check the returns and see if they are callable.
             if decorated_with_property(attr):
-
                 try:
-                    all_returns_are_callable = all(
-                        return_node.callable() or return_node is astroid.Uninferable
-                        for return_node in attr.infer_call_result(node)
-                    )
-                except astroid.InferenceError:
-                    continue
+                    call_results = list(attr.infer_call_result(node))
```

Comment:
Can you place the except directly after this so that the try only wraps what might fail?